easypy/concurrency.py

```diff
@@ -1021,6 +1021,12 @@ def result(self, timeout=None):
             raise self.exc
         return self._result
 
+    def done(self):
```

Comment:
I think `is_done` is more appropriate name. Also, it can be a property

Reply:
If it's `is_done` it should remain a function. If it's a property it should be just `done`.

Reply:
Because "is" is kind of a verb...

Reply:
I disagree, I have seen many properties which start with "is_", makes it clearer that this is a boolean value. I'll leave it for you to decide.